tensorflow/lite/toco/graph_transformations/fuse_binary_into_preceding_affine.cc

```diff
@@ -253,14 +268,22 @@ ::tensorflow::Status FuseBinaryIntoPrecedingAffine::Run(Model* model,
 
   if (preceding_op->type != OperatorType::kConv &&
       preceding_op->type != OperatorType::kFullyConnected &&
-      preceding_op->type != OperatorType::kDepthwiseConv) {
+      preceding_op->type != OperatorType::kDepthwiseConv &&
+      preceding_op->type != OperatorType::kTransposeConv) {
     AddMessageF(
         "Not fusing %s because the preceding %s is not of one of the supported "
         "types",
         LogName(*binary_op), LogName(*preceding_op));
     return ::tensorflow::Status::OK();
   }
 
+  if (preceding_op->type == OperatorType::kTransposeConv &&
```

Comment:
quick question, why did you have to add this if statement?

 i am still tackling some broken tests from an internal team and this is related.

Reply:
I was intend to fused kAdd to kTransposeConv.
Remove if statement, kMul, kSub, kDiv may also be fused to kTransposeConv, which is also good for graph optimise.

Shall I remove it?